src/pages/AnimeSama/main.ts

```diff
@@ -0,0 +1,50 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeSama: pageInterface = {
+  name: 'AnimeSama',
+  domain: 'https://anime-sama.fr',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    return Boolean($('#playerDF').length);
+  },
+  sync: {
+    getTitle(_) {
+      return j.$('#titreOeuvre').text().trim();
```

Comment:
Season 2 will have the same title as season 1. It would be good that everything that is not season 1 should add <title> season <season number> to at least hope it finds the correct entry.